systers_portal/templates/meetup/base.html

```diff
@@ -1,14 +1,14 @@
 {% extends "base.html" %}
 
 {% block title %}
-  {{ object.name }}
+  {{ slug|capfirst }} Systers
```

Comment:
What do you want to achieve with this? 
As far as I remember, the meetup location already contains "Systers" in its title (e.g. "Atlanta Systers").